src/zope/app/apidoc/codemodule/browser/static_menu.pt

```diff
@@ -7,44 +7,37 @@
     <div>
       <span i18n:translate="">Class Finder:</span>
       <span><i i18n:translate="">(Enter partial Python path)</i></span>
-	</div>
+	  </div>
     <form action="#" method="post" name="searchform" >
-	  <br />
+	    <br />
       <input type="text" name="path"
              style="font-size: 80%; width=95%" />
 
-	  <br />
+	    <br />
       <input type="button" name="Find" value="Find"
-	     onClick="
+	           onClick="
                javascript:simplegetSearchResult(document.searchform.path.value)"
              i18n:attributes="value find-button" style="font-size: 80%"/>
-
-<!--
-      <input type="submit" name="SUBMIT" value="Find"
-             i18n:attributes="value find-button" style="font-size: 80%"/>
-  -->
-  </form>
+    </form>
 
     <p style="font-size: 120%">
       <a href="./index.html" target="main" i18n:translate="">Browse
         Zope Source</a>
     </p>
 
-	<div>
+	  <div>
       <span i18n:translate="">Search results:</span><br /><br />
     </div>
 
-    <div tal:define="pyclasses view/findAllClasses"
-         tal:condition="pyclasses" id="resultlist">
-
-      <div tal:repeat="info pyclasses" style="display:block"
-           tal:attributes="id info/counter">
-        <a href="" target="main"
-           tal:attributes="href info/url"
-           tal:content="info/path">
-         /zope/app/Application
-        </a>
-      </div>
+    <div tal:define="classes view/findClasses"
+         tal:condition="classes">
```

Comment:
I don't quite understand why you're dropping `id` attributes here, which may be a sign that I'm a bad person to review zope.app.apidoc's PRs.

(I like documentation generators, I just don't actually use zope.app.apidoc today ... or at any point in the last ten years, I think.)

Reply:
I copied and pasted without checking the differences between `menu.pt` and `static_menu.pt`. I haven't checked to see if the counter IDs are necessary, but the `resultlist` ID is definitely necessary for the JavaScript search function. I also didn't realize that replacing `static_menu.pt`'s `view/findAllClasses` for `menu.pt`'s `view/findClasses` would break the class listing.  
  
All in all, you're a much better reviewer than I am a committer.

Reply:
I didn't even notice the `findClasses` vs `findAllClasses` difference!